packages/web-components/src/components/masthead/masthead-l1.ts

```diff
@@ -231,7 +231,12 @@ class C4DMastheadL1 extends StableSelectorMixin(LitElement) {
       } = this;
 
       // Only act if all the needed elements are present.
-      if (
+      const menuItems = this.l1Data?.menuItems ?? [];
+      if (menuItems.length === 0) {
```

Comment:
Is `menuItems` a mandatory field? Maybe we should put some optional chaining here just in case?

Reply:
It's not mandatory. That's where in the line above, I have the optional chaining for accessing `menuItems`, and use the null coalescing operator to default to an empty array, so that the local variable `menuItems` is always an array.